build/rules.mak

```diff
@@ -75,7 +75,7 @@ $(DRIVER_DOCS_DIR)/%.inc: %.inc.mako $(BUILD_HELPER_SCRIPTS) $(METADATA_FILES)
 
 $(DRIVER_EXAMPLES_ZIP_FILE): $(EXAMPLE_FILES)
 	$(call trace_to_console, "Zipping",$@)
-	$(_hide_cmds)$(call log_command,cd src/$(DRIVER)/examples && zip -u -r -9 $@ * || ([ $$? -eq 12 ] && exit 0) || exit)
+	$(_hide_cmds)$(call log_command,python tools/zip_examples.py --zip-file $@ --src-path src/$(DRIVER)/examples)
```

Comment:
We're no longer trying to create bit-by-bit identical ZIP files. So I think we should go back to the much simpler direct use of `zip` rather than using our own Python script, date blanking, etc.

Reply:
Forgot to make a new PR. This one will be closed and a new one created: